dev-haskell/haskell-lsp/haskell-lsp-0.23.0.0.ebuild

```diff
@@ -1,50 +0,0 @@
-# Copyright 1999-2021 Gentoo Authors
```

Comment:
Can you also add a commit that masks the package explicitly and in the mask explains the alternative?

Reply:
@trofi This package isn't used for anything except haskell-language-server. They basically renamed it on Hackage by deprecating the old name. When HLS is updated it will pull in the new package name. Masking it seems unnecessary.

Reply:
@trofi Done